src/langgraph-platform/hybrid.mdx

```diff
@@ -34,4 +34,50 @@ For information on how to deploy a [LangGraph Server](/langgraph-platform/langgr
 
 <Tip>
   If you would like to deploy to Kubernetes, you can follow the [Hybrid deployment guide](/langgraph-platform/deploy-hybrid).
-</Tip>
\ No newline at end of file
+</Tip>
+
+## Listeners
```

Comment:
I think the "conceptual guide" page (this page) is the appropriate place for this content (as opposed to the `Deploy to Hybrid` page).